src/pyenphase/models/meters.py

```diff
@@ -0,0 +1,44 @@
+"""Model for the Envoy's CT Meters."""
+from __future__ import annotations
```

Comment:
I think the StrEnum is a good change, but I'd do it in its own PR as this PR is already quite large

Reply:
Understand, but some of these are in the new test cases config in test_endpoints already. I can remove the inused ones and re-add them when needed in next feature.

Reply:
I'd open a new PR to add the strenums and than once we merge that one, we can merge the changes back into this one so they shake out here and make it smaller to review

Reply:
Ah, get it. I'll try to create some smaller steps first.